src/components/modal/modal.scss

```diff
@@ -6,6 +6,9 @@
  * @prop --calcite-modal-content-background: Specifies the background color of content placed in the `content` slot.
  * @prop --calcite-modal-content-padding: Specifies the padding of the modal `content` slot.
  * @prop --calcite-modal-scrim-background: Specifies the background color of the modal scrim.
+ * @prop --calcite-modal-width: Specifies a width of the modal. Will never exceed the width of the viewport. Does not apply when `fullscreen` is true.
```

Comment:
I'm reconsidering not having these apply to fullscreen and instead always having them be honored. 

It seems more consistent that the components would adhere to any requested overrides.